code/modules/research/designs/tool_designs.dm

```diff
@@ -170,3 +170,35 @@
 	materials = list(MAT_METAL = 2000, MAT_SILVER = 1500, MAT_PLASMA = 500, MAT_TITANIUM = 1500)
 	category = list("Tool Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
+
+/datum/design/sprinkler
+	name = "Deployable Sprinkler"
+	desc = "A self deployable sprinkler. Use in case of fires."
+	id = "sprinkler"
+	build_path = /obj/item/deployable_sprinkler
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 20000, MAT_GLASS = 10000)
+	category = list("Tool Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING
+
+/datum/design/refill_water
+	name = "Sprinkler Refill (water)"
+	desc = "A sprinkler refill that uses dehydrogen oxyde as coolant."
```

Comment:
dihydrogen oxide

Reply:
Technically, Dihydrogen monoxide. Really, though, you could probably just say water.